projectfiles/Xcode/The Battle for Wesnoth.xcodeproj/project.pbxproj

```diff
@@ -1502,8 +1502,8 @@
 		1058C7A1FEA54F0111CA2CBB /* Cocoa.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = Cocoa.framework; path = System/Library/Frameworks/Cocoa.framework; sourceTree = SDKROOT; };
 		1234567890ABCDEF12345680 /* file_progress.cpp */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.cpp.cpp; path = file_progress.cpp; sourceTree = "<group>"; };
 		1234567890ABCDEF12345681 /* file_progress.hpp */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.cpp.h; path = file_progress.hpp; sourceTree = "<group>"; };
-		000000000000000000000001 /* achievements.cpp */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.cpp.cpp; path = achievements.cpp; sourceTree = "<group>"; };
-		000000000000000000000002 /* achievements.hpp */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.cpp.h; path = achievements.hpp; sourceTree = "<group>"; };
+		000000000000000000000001 /* achievements.cpp */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.cpp.cpp; name = achievements.cpp; path = ../../src/achievements.cpp; sourceTree = SOURCE_ROOT; };
+		000000000000000000000002 /* achievements.hpp */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.cpp.h; name = achievements.hpp; path = ../../src/achievements.hpp; sourceTree = SOURCE_ROOT;  };
```

Comment:
?????

What? Why would you do that? Everything's relative to its group.

…wait, did you maybe forget to add it to the group?

Reply:
I don't even know what a group in xcode is.

Reply:
It's what Visual Studio calls a filter. Basically a container for source files within the project, to just group them together independently of how they're stored on the filesystem.

Using `sourceTree = "<group>"` is the preferred way of referencing files that exist within the source root. Our project links each group to a physical folder, so if the file is relative to the group and placed in the correct group, then it should have no issues finding it.